app_dart/lib/src/request_handlers/get_green_commits.dart

```diff
@@ -54,12 +53,11 @@ class GetGreenCommits extends RequestHandler<Body> {
     final DatastoreService datastore = datastoreProvider(config.db);
     final BuildStatusService buildStatusService = buildStatusProvider(datastore);
     final int commitNumber = config.commitNumber;
-    final int lastCommitTimestamp = DateTime.now().millisecondsSinceEpoch;
 
     final List<String?> greenCommits = await buildStatusService
         .retrieveCommitStatus(
           limit: commitNumber,
-          timestamp: lastCommitTimestamp,
+          timestamp: DateTime.now().millisecondsSinceEpoch,
```

Comment:
You don't need to pass timestamp (by default it'll be the current time).

The reason we want to avoid this is using DateTime.now() will cause flakiness in tests